core/src/validator.rs

```diff
@@ -1056,7 +1056,6 @@ fn report_target_features() {
         // Validator binaries built on a machine with AVX support will generate invalid opcodes
         // when run on machines without AVX causing a non-obvious process abort.  Instead detect
         // the mismatch and error cleanly.
-        #[target_feature(enable = "avx")]
```

Comment:
nits: unrelated change?

Reply:
uh super annoying, my `cargo fmt` keeps deleting this line... thanks!

Reply:
protip: `(unset RUSTFLAGS && cargo +1.45.0 fmt --all)` This way I can live with ever-evolving nightly. :)